/*Create a program to find the power of a number.
Hint => 
Get integer input for two variables - number and power and check for positive integer
Create a result variable with an initial value of 1.
Run a for loop from i = 1 to i <= power. In each iteration of the loop, multiply
*/
import java.util.Scanner;

public class power {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter base number: ");
        int number = sc.nextInt();
        System.out.print("Enter power (positive integer): ");
        int power = sc.nextInt();
        //for loop
		long result = 1;

		for (int i = 1; i <= power; i++) {
			result *= number;
		}
        
		/*
		//while loop
		long result = 1;
		int counter = 0;

		while (counter < power) {
			result *= number;
			counter++;
		}

		*/
		System.out.println(number + " raised to the power " + power + " is: " + result);
    }
}
